internal/engine/upper_test.go

```diff
@@ -2449,34 +2449,6 @@ func TestK8sEventNotLoggedIfNoManifestForUID(t *testing.T) {
 		"should not log event message b/c it doesn't have a UID -> Manifest mapping")
 }
 
-func TestK8sEventDoNotLogNormalEvents(t *testing.T) {
```

Comment:
wait, doesn't this test still apply? do you have test coverage for:
1. don't log "normal" events
2. log if !normal and if `pull` or `pulling` (but presumably not otherwise)?

Reply:
ya, mostly this was testing code that didn't matter anymore, and really needed to be testing the subscriber code. added a new test!